test/e2e/vsphere_utils.go

```diff
@@ -222,6 +223,7 @@ func getVSphereClaimSpecWithStorageClassAnnotation(ns string, storageclass *stor
 	return claim
 }
 
+// func to get pod spec with given volume claim, node selector labels and command
 func getVSpherePodSpecWithClaim(claimName string, nodeSelectorKV map[string]string, command string) *v1.Pod {
 	pod := &v1.Pod{
```

Comment:
here wanted to keep `nodeSelectorKV` optional, so if `nil` value is passed, not setting `pod.Spec.NodeSelector`